CHANGELOG.rst

```diff
@@ -46,6 +46,8 @@ Changed
 - DatetimeInfoExtractor.mappings_provided changed from a dict.keys() object to list so transformer is serialisable. `#258 <https://github.com/lvgig/tubular/pull/258>`_
 - Created BaseNumericTransformer class to support test refactor of numeric file `#266 <https://github.com/lvgig/tubular/pull/266>`_
 - Updated testing approach for LogTransformer `#268 <https://github.com/lvgig/tubular/pull/268>`_
+- Updated `ScalingTransformer` default columns to `["a", "b"]` in `minimal_attribute_dict` to fix test issues.
```

Comment:
I don't think this needs a line in the changelog, its just part of the below point.

Reply:
otherwise happy with changes so happy to approve once we delete this line

Reply:
I have removed the unnecessary changelog entry